RELEASE_NOTES.md

```diff
@@ -151,6 +151,10 @@ Initial support has been added for Red Hat Enterprise Linux 8. Thank you [@pixdr
 
 gem_package now supports installing gems into Ruby 2.6 or later installations.
 
+### scm / git / subversion
+
+scm now supports the path property in favour of destination. However destination will continue to work at this time.
```

Comment:
```suggestion
`scm` resource now supports the `path` property rather than the `destination` property. While currently supported, the `destination` property will be deprecated eventually.
```